packages/web-react/src/components/FileUploader/demo/index.tsx

```diff
@@ -22,14 +21,15 @@ import FileUploaderInputValidationWithIcon from './FileUploaderInputValidationWi
 import FileUploaderInputWithAttachment from './FileUploaderInputWithAttachment';
 import FileUploaderMetaData from './FileUploaderMetaData';
 import FileUploaderMultipleWithValidation from './FileUploaderMultipleWithValidation';
+import FileUploaderSingle from './FileUploaderSingle';
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
     <IconsProvider value={icons}>
-      <DocsSection title="Input">
-        <FileUploaderDefault />
+      <DocsSection title="Input Single">
```

Comment:
Question: Is `Input Single` descriptive enough? 🤷‍♂️ 

```suggestion
      <DocsSection title="Input Single File">
```